src/borg/archiver/diff_cmd.py

```diff
@@ -64,25 +83,61 @@ def build_parser_diff(self, subparsers, common_parser, mid_common_parser):
         from ._common import process_epilog
         from ._common import define_exclusion_group
 
-        diff_epilog = process_epilog(
-            """
-            This command finds differences (file contents, user/group/mode) between archives.
+        diff_epilog = (
+            process_epilog(
+                """
+        This command finds differences (file contents, user/group/mode) between archives.
+
+        A repository location and an archive name must be specified for ARCHIVE1.
+        ARCHIVE2 is just another archive name in same repository (no repository location
+        allowed).
+
+        For archives created with Borg 1.1 or newer diff automatically detects whether
+        the archives are created with the same chunker params. If so, only chunk IDs
+        are compared, which is very fast.
+
+        For archives prior to Borg 1.1 chunk contents are compared by default.
+        If you did not create the archives with different chunker params,
+        pass ``--same-chunker-params``.
+        Note that the chunker params changed from Borg 0.xx to 1.0.
```

Comment:
Just delete 95..102, not relevant any more for borg2 (there will be no old archives).

(this was already outdated before your PR)